iterative/resource_task.go

```diff
@@ -263,14 +266,17 @@ func resourceTaskDelete(ctx context.Context, d *schema.ResourceData, m interface
 
 	task, err := resourceTaskBuild(ctx, d, m)
 	if err != nil {
+		go utils.SendJitsuEvent("task/destroy", err, d)
 		return diagnostic(diags, err, diag.Error)
 	}
 
 	if err := task.Delete(ctx); err != nil {
+		go utils.SendJitsuEvent("task/destroy", err, d)
 		return diagnostic(diags, err, diag.Error)
 	}
 
-	return
+	go utils.SendJitsuEvent("task/destroy", err, d)
```

Comment:
@0x2b3bfa0 @dacbd  with `go` we are not hitting jitsu. No sure if its because we finish prior to the call... When I remove it I can see the logs in my dev server

Reply:
Yes, @DavidGOrtega, it's a bit more involved that adding `go` and forgetting about it. I'll open a cascading pull request after an upcoming meeting.

Reply:
Nah, let me try it

Reply:
👌🏼

Reply:
Opened https://github.com/iterative/terraform-provider-iterative/pull/538 after https://github.com/iterative/terraform-provider-iterative/pull/493#issuecomment-1109800668